src/main/java/pl/codzisnaobiad/imhungry/api/endpoint/RecipeEndpoint.java

```diff
@@ -29,4 +31,9 @@ SearchRecipesResponse searchRecipes(
         return recipeFacade.searchRecipes(recipeRequestModelMapper.toRequestModel(queryParameters));
     }
 
+    @GetMapping("/{id}/information")
+    RecipeInformationResponse recipeInformation(@PathVariable int id) {
```

Comment:
tego path variable lepiej nazwać recipeId i wgłąb wszędzie też

Reply:
done, ale w samym GetMapping zostawiłbym /recipes/id/ingredients bo id jest po członie recipes wiec wiadomo ze chodzi konkretnie o to